packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -714,6 +724,16 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 		return filterValueReset;
 	}
 
+	/**
+	 * Gets the class for a specific row if a rowClassNameFn function has been given as an Input
+	 * @param row - The data object passed to the row.
+	 * @param index - The index of the row.
+	 * @returns The className generated by the rowClassNameFn function
+	 */
+	public getRowClass(row: any, index: number): string {
```

Comment:
Isn't it better to call it "getRowClasses" since it can return multiple classes?